lesson_03/strings/strings.go

```diff
@@ -0,0 +1,49 @@
+package strings
+
+import "fmt"
+
+func Max(inStrings []string) string {
```

Comment:
I wasn't focused on naming and CamelCase - I see no reason. No real use cases - so no design. But I remember that Max would be accessible from other packages and max would not.

Reply:
Yeah, better choose `max`, cause you use it only from package(with no cases in other packages).

Reply:
fixed